tests/jvm/src/test/resources/metac.expect_2.13

```diff
@@ -1729,11 +1729,6 @@ Synthetics:
       a,
       b
     ) == (1, 2))}).flatMap[Tuple6[Int, Int, Int, Int, Tuple4[Int, Int, Int, Int], Tuple4[Int, Int, Int, Int]]]({(b) => orig(List((a, b))).withFilter({(check$ifrefutable$1) => orig()}).withFilter({(local7) => orig((
-      a,
```

Comment:
why did this disappear?

Reply:
I would assume that some internal desugarings changed, but I honestly think these synthetics are not being used currently by anyone.